arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -0,0 +1,954 @@
+          "Namespaces"
+        ],
+        "operationId": "Namespaces_CreateOrUpdate",
+        "description": "Creates/Updates a service namespace. Once created, this namespace's resource manifest is immutable. This operation is idempotent.",
+        "externalDocs": {
+          "url": "https://msdn.microsoft.com/en-us/library/azure/mt639408.aspx"
+        },
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "namespaceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The namespace name."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/NamespaceCreateOrUpdateParameters"
+            },
+            "description": "Parameters supplied to create a Namespace Resource."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Namespace create request accepted",
+            "schema": {
+              "$ref": "#/definitions/NamespaceResource"
+            }
+          },
+          "400": {
+            "description": "Invalid request body"
+            },
+          "500": {
+            "description": "Internal error"
+            }
+          },
+        "x-ms-long-running-operation": true
+      },
+      "delete": {
+        "tags": [
+          "Namespaces"
+        ],
+        "operationId": "Namespaces_Delete",
+        "description": "Deletes an existing namespace. This operation also removes all associated resources under the namespace.",
+        "externalDocs": {
+          "url": "https://msdn.microsoft.com/en-us/library/azure/mt639389.aspx"
+        },
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "namespaceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The namespace name."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "404": {
+            "description": "Namespace does not exist"
+          },
+          "200": {
+            "description": "Namespace delete successful"
+          },
+          "500": {
```

Comment:
Every responses should have a default response. The default response is used for error handling. By default the generated code will deserialize the code, message properties of the error. However, if the service has more detailed information about the error then I would recommend creating an Error model in the definition and reference it as a schema for default response. 

https://github.com/Azure/azure-rest-api-specs/blob/61edf5bb39011e8230853b569cc740200044d92f/arm-cdn/2016-04-02/swagger/cdn.json#L59

https://github.com/Azure/azure-rest-api-specs/blob/61edf5bb39011e8230853b569cc740200044d92f/arm-cdn/2016-04-02/swagger/cdn.json#L2433